media/base/audio_parameters.cc

```diff
@@ -93,6 +97,10 @@ void AudioParameters::Reset(Format format,
   frames_per_buffer_ = frames_per_buffer;
   effects_ = NO_EFFECTS;
   mic_positions_.clear();
+#if defined(SERVICE_OFFLOADING)
+  if (base::ServiceOffloading::IsEnabled())
+    capture_system_audio_ = false;
```

Comment:
Since we have set_capture_system_audio function, I think It would be ``set_capture_system_audio(false)``. and make ``capture_system_audio_`` private. If you do not want to make it private, I don't think we need ``set_capture_system_audio``

Reply:
|capture_system_audio_| is already a private variable.
And a code what you commented is for initialization. In this case, other variables also don't use the 'Setter' function.